src/hooks.tsx

```diff
@@ -106,18 +106,32 @@ export const useAriaId = (id?: string): string => {
     return React.useRef(id || getAriaId()).current;
 };
 
-export const useWindowSize = (): {height: number; width: number} => {
+export const useWindowSize = (): {
+    height: number;
+    width: number;
+    screenHeight: number;
+    screenWidth: number;
+} => {
     const [windowHeight, setWindowHeight] = React.useState<number>(
         typeof window !== 'undefined' ? window.innerHeight : 1200 // Best guess
     );
     const [windowWidth, setWindowWidth] = React.useState<number>(
         typeof window !== 'undefined' ? window.innerWidth : 800 // Best guess
     );
 
+    const [screenHeight, setScreenHeight] = React.useState<number>(
+        typeof window !== 'undefined' ? window.screen.availHeight : 1200
+    );
+    const [screenWidth, setScreenWidth] = React.useState<number>(
+        typeof window !== 'undefined' ? window.screen.availWidth : 800
+    );
+
```

Comment:
no need to put this inside a hook `screen.availHeight` never changes

Reply:
u sure? according to my tests, when you rotate the device, the `availHeight` changes

Reply:
hmm, that's right